src/content/en/updates/2019/08/get-started-with-gpu-compute-on-the-web.md

```diff
@@ -111,17 +114,19 @@ Now let’s see how to copy a GPU buffer to another GPU buffer and read it back.
 
 Since we’re writing in the first GPU buffer and we want to copy it to a second
 GPU buffer, a new usage flag `GPUBufferUsage.COPY_SRC` is required. The second
-GPU buffer is created in an unmapped state with the synchronous
+GPU buffer is created in an unmapped state this time with
 `device.createBuffer()`. Its usage flag is `GPUBufferUsage.COPY_DST |
 GPUBufferUsage.MAP_READ` as it will be used as the destination of the first GPU
 buffer and read in JavaScript once GPU copy commands have been executed.
 
 ```js
 // Get a GPU buffer in a mapped state and an arrayBuffer for writing.
-const [gpuWriteBuffer, arrayBuffer] = device.createBufferMapped({
+const gpuWriteBuffer = device.createBuffer({
+  mappedAtCreation: true,
   size: 4,
   usage: GPUBufferUsage.MAP_WRITE | GPUBufferUsage.COPY_SRC
 });
+const arrayBuffer = gpuWriteBuffer.getMappedRange();
```

Comment:
ditto, this call to `getMappedRange` should be removed.

Reply:
We need it to write to it...

Reply:
Nvm :)